src/importer.js

```diff
@@ -0,0 +1,50 @@
+const fs = require('fs');
+const EventEmitter = require('events').EventEmitter;
+const csvjson = require('csvjson');
+
+const checkedFiles = [];
+
+class Importer extends EventEmitter {
+  constructor(dirWatcher, path) {
+    super();
+    // sync call:
+    // dirWatcher.on('data change', () =>
+    //   this.importSync(path)
+    // );
+    // ---------------
+    // async call:
+    dirWatcher.on('data change', () =>
+      this.import(path)
+        .then(data => console.log(data))
+        .catch(err => console.log(err)));
+  }
+
+  import(path) {
```

Comment:
It can be easily changed to async/await method when you will use Promise-based fs API